### PR TITLE
Improve ALB capabilities

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

**Comment by katcipis** [comment]
Now we can add/remove load balancers dynamically from NICs (not only on creation), also we can do this with load balancers residing on another resource group (covered only on example).

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ integration_timeout=50m
 examples_timeout=90m
 all_timeout=90m
 logger=file
-parallel=30 #Explore I/O parallelization
+parallel=60 #Explore I/O parallelization
```
Comment (lborguetti) [inline Makefile]: :scream:

File: azure/lb.sh [hunk 2]
```diff
@@ -82,7 +82,7 @@ fn azure_lb_frontend_ip_set_subnet_vnet_name(instance, subnetvnetname) {
 }
 
 fn azure_lb_frontend_ip_create(instance) {
-	(azure network lb frontend-ip create $instance)
+	azure network lb frontend-ip create $instance
```
Comment (lborguetti) [inline azure/lb.sh]: I think we can change to Az 2.0 here!

File: azure/lb.sh [hunk 3]
```diff
@@ -97,7 +97,7 @@ fn azure_lb_frontend_ip_delete(name, group, lbname) {
 # ADDRESS POLL functions
 
 fn azure_lb_addresspool_create(name, group, lbname) {
-	out           <= (
+	out <= (
```
Comment (lborguetti) [inline azure/lb.sh]: Same here :point_up:

Comment (lborguetti) [inline azure/lb.sh]: Same here :point_down:

File: azure/lb.sh (hunk 2 above)
Comment (katcipis) [reply azure/lb.sh]: I was on the ASAP mode =). Perhaps on a future refactoring.

**Pull request merged**